content/create-an-addon/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Create an Addon'
+description: 'Learn how to build your own addons that will super charge your development'
+heroDescription: 'Addons give you superpowers to extend Storybook, automate workflows, and integrate with your favorite tools. This guide shows you how to create an addon.'
+overview: 'While the Storybook community offers over 250 addons, you can also build one tailored to your specific needs. This guide introduces you to the Addon Kit and APIs, along the way you will build an addon from scratch.'
+order: 4
+themeColor: '#FF4785'
+codeGithubUrl: 'https://github.com/chromaui/learnstorybook-addon-code'
+heroAnimationName: null
+toc:
+  [
+    'introduction',
+    'getting-started',
+    'register-an-addon',
+    'track-state',
+    'decorators',
+    'preset',
+    'add-to-catalog',
+    'conclusion',
+  ]
+coverImagePath: '/guide-cover/create-an-addon.svg'
+thumbImagePath: '/guide-thumb/create-an-addon.svg'
+contributorCount: '+3'
+authors:
+  [
+    {
+      src: 'https://avatars2.githubusercontent.com/u/42671',
+      name: 'Varun Vachhar',
+      detail: 'Storybook DX',
+    },
+  ]
+contributors: []
+twitterShareText: "I’m learning how to build a Storybook addon! They're great for customizing Storybook to fit your UI development workflow."
+---
+
+<h2>What you'll build</h2>
+
+<div class="badge-box">
+  <div class="badge">
+    <img src="/frameworks/logo-react.svg"> <a href="/create-an-addon/react/en/introduction/"> React</a>
```

Comment:
Let's remove the link here – we don't have consistent behavior for badge links which ends up being strange.

```suggestion
    <img src="/frameworks/logo-react.svg"> React
```